modules/my_health/app/controllers/my_health/v1/allergies_controller.rb

```diff
@@ -6,16 +6,12 @@ class AllergiesController < MrController
       def index
         patient_id = params[:patient_id]
         resource = client.list_allergies(patient_id)
-        raise Common::Exceptions::InternalServerError if resource.blank?
-
         render json: resource.to_json
       end
 
       def show
         allergy_id = params[:id].try(:to_i)
         resource = client.get_allergy(allergy_id)
-        raise Common::Exceptions::InternalServerError if resource.blank?
```

Comment:
Does there need to be any other error handling here? What does `resource.to_json` return if `resource` is blank?

Reply:
My thought was to handle these errors further down the stack, in the client code: https://github.com/department-of-veterans-affairs/vets-api/blob/33e15033cdf861203b7420bc4d5da6e75d80134f/lib/medical_records/client.rb#L110